.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
po co to?

Reply:
Moj lokalny config ktory sie zabral bo nie byl wykluczony w gitignore